udev/99-stratisd.rules

```diff
@@ -0,0 +1 @@
+ENV{DM_NAME}=="stratis-1-[0-9a-f]*-thin-fs-[0-9a-f]*", PROGRAM="/usr/lib/udev/stratis_uuids_to_names $env{DM_NAME} $env{ACTION}", SYMLINK+="stratis/%c{1}/%c{2}"
```

Comment:
What is the concern here? I'm unclear on what potential problems you see this causing. And what kind of indirection would you see being helpful? A symlink? Use of a variable provided by udev internally? I think that because we are providing that executable just like we are providing the udev rule, there is no ambiguity in the absolute path that we're providing so if the package is installed, one should be able to access the other without a problem.

Reply:
The macro `%{_libdir}` evaluates to `/usr/lib64` on an x86-64 system, yet the correct path to use for this file is `/usr/lib/udev`.  This may be only specific to the spec file, for packaging, but it does call into question why there's a `%{_udevrulesdir}` macro (which resolves to `/usr/lib/udev/rules.d`), and not one for `/usr/lib/udev`.

Reply:
@bgurney-rh has hit on the area of my concern; it's unforeseen interactions between Fedora/RHEL packaging and our hard-coding. Right now there are none; because the spec file also hard codes the path: https://github.com/stratis-storage/ci/pull/106/commits/b82400732e14201f10bf28b30f0cb32b9086a123. So we can proceed.

Reply:
I'm going to test removing the absolute path and using a relative one instead, and I'll report back.

Reply:
This works.